SPECS/avahi/avahi.spec

```diff
@@ -0,0 +1,1083 @@
+mkdir -p %{buildroot}%{_localstatedir}/lib/avahi-autoipd
+
+# remove the documentation directory - let % doc handle it:
+rm -rfv %{buildroot}%{_datadir}/%{name}-%{version}
+
+# Make /etc/avahi/etc/localtime owned by avahi:
+mkdir -p %{buildroot}%{_sysconfdir}/avahi/etc
+touch %{buildroot}%{_sysconfdir}/avahi/etc/localtime
+
+# fix bug 197414 - add missing symlinks for avahi-compat-howl and avahi-compat-dns-sd
+%if %{WITH_COMPAT_HOWL}
+ln -s avahi-compat-howl.pc  %{buildroot}/%{_libdir}/pkgconfig/howl.pc
+%endif
+%if %{WITH_COMPAT_DNSSD}
+ln -s avahi-compat-libdns_sd.pc %{buildroot}/%{_libdir}/pkgconfig/libdns_sd.pc
+ln -s avahi-compat-libdns_sd/dns_sd.h %{buildroot}/%{_includedir}/
+%endif
+
+# unpackaged files
+rm -fv  %{buildroot}%{_datadir}/applications/{bssh,bvnc}.desktop
+rm -fv  %{buildroot}%{_datadir}/avahi/interfaces/avahi-discover.ui
+
+rm -fv %{buildroot}%{_sysconfdir}/rc.d/init.d/avahi-daemon
+rm -fv %{buildroot}%{_sysconfdir}/rc.d/init.d/avahi-dnsconfd
+
+%find_lang %{name}
+
+
+%check
+
+%pre
+getent group avahi >/dev/null || groupadd -f -g 70 -r avahi
+if ! getent passwd avahi > /dev/null ; then
+  if ! getent passwd 70 > /dev/null ; then
+    useradd -r -l -u 70 -g avahi -d %{_localstatedir}/run/avahi-daemon -s %{_sbindir}/nologin -c "Avahi mDNS/DNS-SD Stack" avahi
+  else
+    useradd -r -l -g avahi -d %{_localstatedir}/run/avahi-daemon -s %{_sbindir}/nologin -c "Avahi mDNS/DNS-SD Stack" avahi
+  fi
+fi
+exit 0
+
+%post
+%{?ldconfig}
+%{_bindir}/dbus-send --system --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig >/dev/null 2>&1 || :
+if [ "$1" -eq 1 -a -s %{_sysconfdir}/localtime ]; then
+        %{_bindir}/cp -cfp %{_sysconfdir}/localtime %{_sysconfdir}/avahi/etc/localtime >/dev/null 2>&1 || :
+fi
+%systemd_post avahi-daemon.socket avahi-daemon.service
+
+%preun
+%systemd_preun avahi-daemon.socket avahi-daemon.service
+
+%postun
+%{?ldconfig}
+%systemd_postun_with_restart avahi-daemon.socket avahi-daemon.service
+
+%pre autoipd
+getent group avahi-autoipd >/dev/null || groupadd -f -g 170 -r avahi-autoipd
+if ! getent passwd avahi-autoipd > /dev/null ; then
+  if ! getent passwd 170 > /dev/null; then
+    useradd -r -u 170 -l -g avahi-autoipd -d %{_localstatedir}/lib/avahi-autoipd -s %{_sbindir}/nologin -c "Avahi IPv4LL Stack" avahi-autoipd
+  else
+    useradd -r -l -g avahi-autoipd -d %{_localstatedir}/lib/avahi-autoipd -s %{_sbindir}/nologin -c "Avahi IPv4LL Stack" avahi-autoipd
+  fi
+fi
+exit 0
+
+%post dnsconfd
+%systemd_post avahi-dnsconfd.service
+
+%preun dnsconfd
+%systemd_preun avahi-dnsconfd.service
+
+%postun dnsconfd
+%systemd_postun_with_restart avahi-dnsconfd.service
+
+%ldconfig_scriptlets glib
+
+%ldconfig_scriptlets compat-howl
+
+%ldconfig_scriptlets compat-libdns_sd
+
+%ldconfig_scriptlets libs
+
+%ldconfig_scriptlets ui
+
+%ldconfig_scriptlets gobject
+
+%files -f %{name}.lang
+%doc docs/* avahi-daemon/example.service avahi-daemon/sftp-ssh.service avahi-daemon/ssh.service
```

Comment:
Do you know, if there's a license we could install with `%license`?

Reply:
There is already a license packaged this way in the `libs` subpackage, which is a transitive dependency of each of the other subpackages and the main package.

Reply:
Ah, sorry, I missed it.:/